org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
+rename-icon: qutebrowser
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+cleanup:
+- "/lib/debug"
```

Comment:
Preferably put this under the module responsible for the files, to improve portability/readability.